src/java/picard/vcf/GenotypeConcordance.java

```diff
@@ -281,6 +286,17 @@ else if (callVariantContextType == INDEL) {
             progress.record(variantContextForLogging.getContig(), variantContextForLogging.getStart());
         }
 
+        //snp counter add in X number of missing-missing hom ref's (truth and call state)
+        //missing missing is total interval size minus number of iterations in while loop
+
+        if(MISSING_SITES_HOM_REF && usingIntervals){
+            addMissingMissingSites(snpCounter.getCounterSize(), intervalBaseCount, snpCounter);
```

Comment:
I see it throughout so guessing the answer is 'yes' but double checking that `Missing`